### PR TITLE
Including PGOPTIONS="-c gp_role=utility" to use utility mode when inv…

### PR DESCRIPTION
**Pull request by Annu149**
…oking pg_isready.

pg_isready was generating fatal message on primary segments to use utiltily mode for direct connections to primary segments,included PGOPTIONS="-c gp_role=utility" with pg_isready command to avoid the fatal message.

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

**Review by bhuvnesh2703 — COMMENTED** [review]
Could we add a test case for it?

**Comment by kaknikhil** [comment]
+1 for adding a behave test
Some observations:
1. Commit title could be shorter (somewhere between 50-72 chars should work)
2. Since the commit title will be short, we should make the commit message a bit more detailed.

File: gpMgmt/test/behave/mgmt_utils/steps/mgmt_utils.py [hunk 1]
```diff
@@ -698,6 +698,26 @@ def impl(context, command, out_msg, num):
         raise Exception("Expected %s to occur %s times. Found %d. stdout: %s" % (out_msg, num, count, msg_list))
 
 
+@then('the pg_log files on primary segments should not contain "{msg}"')
```
[the remark this answers is not in the document]
Comment (Annu149) [reply gpMgmt/test/behave/mgmt_utils/steps/mgmt_utils.py:701]: Thanks Ashwin for this sql function, will use this.

**Review by ashwinstar — APPROVED** [review]
LGTM

**Review by pobbatihari — APPROVED** [review]

**Pull request merged**